doc/source/whatsnew/v1.2.0.rst

```diff
@@ -410,6 +410,7 @@ Groupby/resample/rolling
 - Bug in :meth:`Rolling.sum()` returned wrong values when dtypes where mixed between float and integer and axis was equal to one (:issue:`20649`, :issue:`35596`)
 - Bug in :meth:`Rolling.count` returned ``np.nan`` with :class:`pandas.api.indexers.FixedForwardWindowIndexer` as window, ``min_periods=0`` and only missing values in window (:issue:`35579`)
 - Bug where :class:`pandas.core.window.Rolling` produces incorrect window sizes when using a ``PeriodIndex`` (:issue:`34225`)
+- Bug in :meth:`Groupby` dropped ``nan`` groups from result with ``dropna=False`` when grouping over a single column (:issue:`35646`, :issue:`35542`)
```

Comment:
DataFrame.groupby; does this PR also fix Series.groupby?

Reply:
Did not know that this issue exists there too. Unfortunately it is not fixed. I will look into this. Do you know any issues about this? Changed the whatsnew

Reply:
Sorry, that was wrong. Series.groupby worked before.